ocs_ci/ocs/utils.py

```diff
@@ -830,7 +830,13 @@ def collect_ocs_logs(dir_name, ocp=True, ocs=True, mcg=False):
             '/usr/bin/gather_service_logs worker'
         )
     if mcg:
-        collect_noobaa_db_dump(log_dir_path)
+        mcg_db_collected = False
+        while not mcg_db_collected:
```

Comment:
Hope this is just for some debugging purpose as I see DNM prefix :) I am adding WIP prefix to title as this really prevent us to merge.

Reply:
+1